pkg/storage/crd/options.go

```diff
@@ -0,0 +1,27 @@
+package crd
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/generic"
+	restclient "k8s.io/client-go/rest"
+)
+
+// Options is the set of options to create a new TPR storage interface
+type Options struct {
```

Comment:
Will need a Validate() method to call per upstream guidance. As part should maybe call the RESTOptions.Validate() if it exists.